phpcs.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="phpList Coding Standard">
+    <description>
+        This standard requires PHP_CodeSniffer >= 3.6.0.
+    </description>
+
+    <arg name="colors"/>
+    <arg name="extensions" value="php"/>
+
+    <!-- The complete PSR-12 ruleset -->
+    <rule ref="PSR12">
```

Comment:
It would also be useful to add the [`PHPCompatibility`](https://github.com/PHPCompatibility/PHPCompatibility) sniffs to make sure that PHP 5.6 features are used.

Reply:
This probably will be a bit more work to get those to run with a PHIVE-based installation. Please let's do this in a separate PR. (I can do that.)